lib/node_modules/@stdlib/math/base/special/boxcox/test/fixtures/python/runner.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+#
+# @license Apache-2.0
+#
+# Copyright (c) 2018 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generate fixtures."""
+
+from os import path
+import json
+from scipy.special import boxcox
+import numpy as np
+
+# Get the file path.
+FILE = path.realpath(__file__)
+
+# Get the file's directory.
+DIR = path.dirname(FILE)
+
+
+def generate(x, y, filename):
+    """Generate fixture data and write to file.
+
+    # Arguments
+
+    * `x`: domain
+    * `y`: domain
+    * `name::str`: filename of the output file
+
+    # Examples
+
+    ```python
+    python> x = np.arange(0.25, 5, 0.25)
+    python> y = np.arange(0.25, 5.0, 0.25)
+    python> generate(x, y, './data.json')
+    ```
+    """
+    z = boxcox(x, y)
+    data = dict(
+        x=x.tolist(),
+        y=y.tolist(),
+        expected=z.tolist()
+    )
+
+    filepath = path.join(DIR, filename)
+
+    with open(filepath, 'w') as out:
+        json.dump(data, out)
+
+
+def main():
+    """Generate fixture data."""
+    x = np.arange(0.25, 5.0, 0.25)
```

Comment:
I can see that being an issue. Is there a place that I could take a look at a testing construct like you are discussing?

Reply:
@justin1dennison See this [file](https://github.com/stdlib-js/stdlib/blob/e3d5637c15e968357b08a251f96739480b8b5aa6/lib/node_modules/%40stdlib/math/base/special/exp10/test/fixtures/python/runner.py). While not using `random`, we try to sample different bit patterns by ensuring that increments are not "nice", which means, in this case, instead of 1000 values, generating 1007 values, thus making the increment (hopefully) an irrational number, or at least a number which does not have a "regular" underlying bit pattern.